src/locales/en/fields.yml

```diff
@@ -256,3 +256,10 @@ date-month:
   label: Month
 date-year:
   label: Year
+govuk:
+  languageToggle:
+    ariaLabel: Dewis iaith
```

Comment:
This aria-label probably wants to be english?

Reply:
Are you sure? In this scenario they're expecting english, including this accessibility label for screen readers, I imagine